test/eco/colsystem.result

```diff
@@ -0,0 +1,6 @@
+load-failure: [Errno 2] No such file or directory: '/Users/ssbarnea/.cache/ansible-lint-eco/colsystem/tests/ansible-lint.yml' (filenotfounderror)
```

Comment:
Is this going to cause false positives when running in other people's medicines?